### PR TITLE
adds the index file

### PR DESCRIPTION
**Pull request by willrpike**
Hey Worked really hard on this what do you think!!!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (eawest2) [inline index.html:6]: //Why are you doing this? What have you even done?

Comment (willrpike) [inline index.html:6]: My understanding is that we agree'd not to support IE Edge, and therefor shouldn't bloat our software with their meta tags!!

Comment (willrpike) [reply index.html:6]: who cares...

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (eawest2) [inline index.html:2]: HTML is a dead format, we should move on to Dreamweaver.

**Pull request merged**